wip/jstachio-1.0.0.buildspec

```diff
@@ -0,0 +1,21 @@
+groupId=io.jstach
+artifactId=jstachio
+version=1.0.0
+
+display=${groupId}:${artifactId}
+
+gitRepo=https://github.com/jstachio/${artifactId}.git
+gitTag=v${version}
+
+tool=mvn-3.9.2
+jdk=17
+newline=lf
+
+# Unfortunately one of the Ant calls to update jstachio-apt jar did not normalize the timestamp on UTC
+# This will be fixed in the next version
+command="bin/rebuild.sh -Duser.timezone=America/New_York"
+
+# Location of the buildinfo file generated during rebuild to record output fingerprints
+buildinfo=api/jstachio/target/${artifactId}-${version}.buildinfo
```

Comment:
define `buildinfo=target/jstachio-maven-parent-${version}.buildinfo` to get the aggregated full build output for 31 files, not only 1 random module with 3 files